code/modules/surgery/organs/tongue.dm

```diff
@@ -92,6 +92,84 @@
 		message = lizard_eckS.Replace(message, "ECKS$1")
 	speech_args[SPEECH_MESSAGE] = message
 
+/obj/item/organ/tongue/lizard/silver
+	name = "silver tongue"
+	desc = "A genetic branch of the high society Silver Scales that gives them their silverizing properties. To them, it is everything, and society traitors have their tongue forcibly revoked. Oddly enough, it itself is just blue."
+	icon_state = "silvertongue"
+	actions_types = list(/datum/action/item_action/organ_action/statue)
+
+/datum/action/item_action/organ_action/statue
+	name = "Become Statue"
+	desc = "Become an elegant silver statue. Its durability and yours are directly tied together, so make sure you're careful."
+	var/ability_cooldown = 0
+	var/obj/structure/statue/custom/statue
+
+/datum/action/item_action/organ_action/statue/New(Target)
+	. = ..()
+	statue = new(null)
+	RegisterSignal(statue, COMSIG_PARENT_QDELETING, .proc/statue_destroyed)
+
+/datum/action/item_action/organ_action/statue/Destroy()
+	UnregisterSignal(statue, COMSIG_PARENT_QDELETING)
+	QDEL_NULL(statue)
+	. = ..()
+
+
+/datum/action/item_action/organ_action/statue/Trigger()
+	. = ..()
+	if(!iscarbon(owner))
+		to_chat(owner, "<span class='warning'>Your body rejects the powers of the tongue!</span>")
+		return
+	var/mob/living/carbon/becoming_statue = owner
+	if(becoming_statue.health < 1)
```

Comment:
what if the statue was reduced to 0.5 health before you turned back? would you just permanently be unable to turn into a statue again?

Reply:
We talked on discord, not an issue